roles/zabbix_agent/tasks/main.yml

```diff
@@ -21,7 +21,9 @@
 - name: Stopping Install of Invalid Version
   fail:
     msg: Zabbix version {{ zabbix_agent_version }} is not supported on {{ ansible_distribution }} {{ ansible_distribution_major_version }}
-  when: not zabbix_valid_version
+  when:
+    - not zabbix_valid_version
+    - ansible_os_family != "Windows"
```

Comment:
Why do we need this additional condition? this should work for all OS's.

Reply:
Because I missed that line lol.  One minute